proto/minder/v1/minder.proto

```diff
@@ -1462,17 +1501,32 @@ message UpdateDataSourceResponse {
     DataSource data_source = 1;
 }
 
-message DeleteDataSourceRequest {
+message DeleteDataSourceByIdRequest {
```

Comment:
uhm...you could leave this as-is and just add a `name` parameter. That way you don't need to create new protobuf messages nor new APIs.

Reply:
I thought the same but then decided it might be better to abstract them into separate concerns in case we need the potential flexibility in the future. Still, if you insist, I can do it that way 😄

Reply:
Either way is fine for me. Just thought I'd make your life easier haha. But we can separate them as you suggest if you prefer.